src/qml/qgismobileapp.qml

```diff
@@ -1703,6 +1706,23 @@ ApplicationWindow {
 
     MenuSeparator { width: parent.width }
 
+    MenuItem {
+      id: addBookmarkItem
+      text: qsTr( "Add Bookmark at Coordinates" )
+      height: 48
+      font: Theme.defaultFont
```

Comment:
Never checked... Can we override that globally somehow?

Reply:
AFAIK, no.